webapp/server/config.js

```diff
@@ -177,26 +170,20 @@ const config = {
         LOG_FILE_MAX_QUANTITY: process.env.LOG_FILE_MAX_QUANTITY || "14d",
     },
     PROJECTS: {
-        // TODO: Document this.
+        // Directory to store workflow results.
         BASE_DIR: process.env.PROJECTS_BASE_DIR || path.join(IO_BASE_DIR, "projects"),
         // Number of days for which the system will preserve a project after a user opts to delete it.
         PROJECT_DELETION_GRACE_PERIOD_DAYS: makeIntIfDefined(process.env.PROJECT_DELETION_GRACE_PERIOD_DAYS) || 7,
     },
-    SPECIES: {
-        // TODO: This is based upon the environment variable originally named `SPECIES_JSON`,
-        //       which was referenced in the code base, but was not defined or documented in
-        //       the former `server-end-prod` template.
-        JSON_FILE_PATH: process.env.SPECIES_JSON_FILE_PATH,
-        // TODO: This is based upon the environment variable originally named `SPECIES_TREE_JSON`,
-        //       which was referenced in the code base, but was not defined or documented in
-        //       the former `server-end-prod` template.
-        TREE_JSON_FILE_PATH: process.env.SPECIES_TREE_JSON_FILE_PATH,
```

Comment:
I am concerned about this being deleted while it is still being referenced in the application code.

https://github.com/microbiomedata/nmdc-edge/blob/fb1e72ad77150a894dd74f1a4ff6953634f6b29a/webapp/server/routes/auth-api/user.js#L741

Reply:
Hi @yxu-lanl, I saw that you marked this conversation as resolved, but I still see the reference to this config variable (the config variable is being deleted in this branch):

https://github.com/microbiomedata/nmdc-edge/blob/fb1e72ad77150a894dd74f1a4ff6953634f6b29a/webapp/server/routes/auth-api/user.js#L741

If someone were to run that code, `config.SPECIES` would be `undefined` (since it is being deleted in this branch) and JavaScript would end up trying to access a property of `undefined` (which raises an exception, as shown below).

![image](https://github.com/microbiomedata/nmdc-edge/assets/134325062/fa920a3a-66cd-4d5d-a85d-ba4a45b9d999)

Reply:
Oops! I just saw you removed the code that references it. Thanks! I'll resolve this thread.